packages/wdio-runner/tests/index.test.js

```diff
@@ -3,10 +3,14 @@ import fs from 'fs'
 import { attach } from 'webdriverio'
 import WDIORunner from '../src'
 
-jest.mock('fs')
+const writeFile = fs.writeFile
```

Comment:
What is the reason for this change?

Reply:
the reason is that we not can't mock `fs` now before `webdriverio/src/utils.js` has been called.
https://github.com/webdriverio/webdriverio/pull/4229/files#diff-fe019f9e87ddc9284183a1356b00a163

Reply:
I don't understand, why is that? Afaik doing 

```js
const writeFile = fs.writeFile
// ...
    beforeAll(() => {
        fs.writeFile = jest.fn()
    })
```
is the same as if we would do:

```js
jest.mock('fs')
```
right away!

Reply:
@christian-bromann taking a look.